### PR TITLE
Homework formatting code Bozhkov

### PR DESCRIPTION
**Pull request by evans2014**

### CODE REVIEW

File: formatting-code/ib/JavaScript-code.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+var txt = "";
+var pX = 0;
+var pY = 0;
+document.onmousemove = mouseMove;
+if (b == "Netscape") {
+	document.captureEvents(Event.MOUSEMOVE)
+}
+
+function mouseMove(evn) {
+	if (b == "Netscape") {
```
Comment (asparuh-krastev) [inline formatting-code/ib/JavaScript-code.js:17]: Tab. It should be 2 or 4 spaces. Everywhere.

File: formatting-code/ib/JavaScript-code.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	addScroll = true;
+}
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+document.onmousemove = mouseMove;
+if (b == "Netscape") {
+	document.captureEvents(Event.MOUSEMOVE)
+}
```
Comment (asparuh-krastev) [inline formatting-code/ib/JavaScript-code.js:14]: I think the above code should be separated with newlines on few places

File: formatting-code/ib/JavaScript-code.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+function PopTip() {
+	if (b == "Netscape") {
+		theLayer = eval('document.layers[\'ToolTip\']');
+		if ((pX + 120) > window.innerWidth) {
+			pX = window.innerWidth - 150;
+		}
+		theLayer.left = pX + 10;
+		theLayer.top = pY + 15;
+		theLayer.visibility = 'show';
+	} else {
```
Comment (asparuh-krastev) [inline formatting-code/ib/JavaScript-code.js:45]: "else {" should be on a new line. Like this:
}
else {

File: formatting-code/ib/Manager.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<?php namespace League\CLImate\Settings;
```
Comment (asparuh-krastev) [inline formatting-code/ib/Manager.php:1]: Should be:
<?php

namespace League\CLImate\Settings;

File: formatting-code/ib/Manager.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+	public function get( $key ) {
+		if ( array_key_exists( $key, $this->settings ) ) {
+			return $this->settings[ $key ];
+		}
+
+		return false;
+	}
+
+	protected function getPath( $name ) {
```
Comment (asparuh-krastev) [inline formatting-code/ib/Manager.php:28]: First all public functions, then protected.

File: formatting-code/ib/JavaScript-code.js (hunk 1 above)
Comment (up2pk) [inline formatting-code/ib/JavaScript-code.js:17]: Yup, 4 spaces.

**Pull request merged**